sdk/src/manifest.rs

```diff
@@ -1022,12 +1022,34 @@ impl Manifest {
     }
 
     /// Embed a signed manifest into a stream using a supplied signer.
-    /// returns the bytes of the new asset
+    ///
+    /// Returns the bytes of the new asset.
+    ///
+    /// We plan to deprecate this; please use embed_to_stream instead.
```

Comment:
Should we formally mark this as deprecated?

See https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute.